saleor/core/utils/__init__.py

```diff
@@ -153,3 +153,14 @@ def prepare_unique_attribute_value_slug(attribute: "Attribute", slug: str):
         "slug", flat=True
     )
     return prepare_unique_slug(slug, value_slugs)
+
+
+def truncate_msg(message: str, max_length: int) -> str:
```

Comment:
Should we use `truncatechars` instead of creating a new function for truncating?